components/typography/stories/template.js

```diff
@@ -129,6 +130,7 @@ const RegularBoldItalicGroup = (args, context) => Container({
 			content: [
 				{
 					content: `Regular ${args.semantics} text`,
+					customStyles: {"margin-block-start": "0"},
```

Comment:
Some spacing updates also got added in with this lang attribute commit! I didn't love the uneven/uncollapsed spacing before headings, but can leave these custom styles off if there's a good reason to do so!